### PR TITLE
feat: Add `StorageManager`

### PR DESCRIPTION
**Pull request by KonTra7cf**

### CODE REVIEW

**Review by maciejopalinski — CHANGES_REQUESTED** [review]
Yup it is almost perfect now. Just fix your formatting (Ctrl + Shift + i in vscode) and export your class as follows:
```
export default new StorageManager();
```
And remove export from your class.

We should export an instance of class because we don't want to create a new instance every time we use our class. That would be memory intensive.

Also you can just commit on your old pull request, but you closed it. There is no need for creating a new one.

**Review by maciejopalinski — APPROVED** [review]

File: src/services/StorageManager.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+class StorageManager {
+  addItem(name, myObj) {
+    let myObj_serialized = JSON.stringify(myObj);
```
Comment (Veryyapeee) [inline src/services/StorageManager.js]: const

File: src/services/StorageManager.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    localStorage.setItem(name, myObj_serialized);
+  } 
+  removeItem(name) {
+    localStorage.removeItem(name);
+  }
+  getItem(name) {
+    let myObj_deserialized = JSON.parse(localStorage.getItem(name));
```
Comment (Veryyapeee) [inline src/services/StorageManager.js]: const

**Review by Veryyapeee — APPROVED** [review]
Approve is here, but pls fix this minor changes before merge :P

**Comment by Veryyapeee** [comment]
lgtm 8), you can merge now

**Pull request merged**